topi/python/topi/generic/nn.py

```diff
@@ -143,6 +159,41 @@ def schedule_depthwise_conv2d_nhwc(outs):
     """
     return _default_schedule(outs, False)
 
+@tvm.target.generic_func
+def schedule_bitserial_conv2d_nchw(outs):
+    """Schedule for qconv2d_nchw
```

Comment:
qconv2d->bitserial_conv2d, do a search over qconv2d and replace by bitserial_conv2d